mypyc/test-data/run-weakref.test

```diff
@@ -0,0 +1,25 @@
+# Test cases for weakrefs (compile and run)
+
+[case testWeakrefRef]
+import pytest  # type: ignore [import-not-found]
```

Comment:
This is not how the run tests work. Instead you should have
```
[file driver.py]
from native import test_weakref_ref, ...
```
and then actually call the functions. There are plenty of examples in other `run-xxx.test` files.

Reply:
Oh good catch, fixed!